src/Aspire.ProjectTemplates/templates/aspire-empty/9.0/AspireApplication.1.ServiceDefaults/Extensions.cs

```diff
@@ -93,7 +94,8 @@ public static TBuilder AddDefaultHealthChecks<TBuilder>(this TBuilder builder) w
     {
         builder.Services.AddHealthChecks()
             // Add a default liveness check to ensure app is responsive
-            .AddCheck("self", () => HealthCheckResult.Healthy(), ["live"]);
+            .AddCheck("self", () => HealthCheckResult.Healthy(), ["live"])
+            .AddApplicationStatus();
```

Comment:
Sorry for not being clear. I think what we want here is for the application status healthcheck to be added with the `'live"` tag, not that we want to keep both the original check and application status.

Reply:
Thanks for clarifying, added with `"live"` tag